packages/components/user/Username.tsx

```diff
@@ -15,39 +15,45 @@ export const Username: FC<{
   userId: string | undefined;
   addressLength?: number;
   textStyle?: StyleProp<TextStyle>;
-}> = memo(({ userId, addressLength, textStyle }) => {
-  const [network, userAddress] = parseUserId(userId);
-  const { primaryAlias } = useNSPrimaryAlias(userId);
-  const {
-    data: { allValidators },
-  } = useValidators(network?.id); // FIXME: this is ineficient
-  const validator = allValidators?.find((val) => val.address === userAddress);
-  if (isUserValidator(userId) && validator) {
+  namedColor?: ColorValue;
+  anonColor?: ColorValue;
+  prefix?: string;
+}> = memo(
```

Comment:
I see that we'd like to memoize component rendering, but if we use inline style which is object like ` {"attr": value} `,  for textStyle,  it will re-render this component every circle :D so `memo` here is useless, IMHO.

Reply:
there is lot of uses where the style is constant